mp/src/game/shared/momentum/mom_stickybomb.cpp

```diff
@@ -325,4 +328,11 @@ void CMomStickybomb::VPhysicsCollision(int index, gamevcollisionevent_t *pEvent)
         m_vecImpactNormal.Negate();
     }
 }
+
+void CMomStickybomb::VPhysicsUpdate(IPhysicsObject *pPhysics)
+{
+    BaseClass::VPhysicsUpdate(pPhysics);
+    m_bCanExplode.Set(!CNoGrenadesZone::IsInsideNoGrenadesZone(this) && !m_bFizzle);
+}
```

Comment:
Can this instead be done inside the nogrenades trigger `OnStartTouch`/`OnEndTouch`? You may have to play around with the entity collision flags to get them to trigger things but I think this may be a better way than having every sticky constantly for-loop the triggers.

Reply:
Sure can :)